packages/pluggableWidgets/color-picker-web/src/ColorPicker.editorConfig.ts

```diff
@@ -0,0 +1,5 @@
+import { Properties } from "@mendix/piw-utils-internal";
+
+export function getProperties(__values: any, defaultProperties: any): Properties {
```

Comment:
Is there a reason you didn't add the types for both function parameters? If not, please add them. We already know that they will be `ColorPickerPreviewProps` and `Properties`, so might as well add them.

Reply:
Nah, I just forgot about it. But one question, If I don't have any kind of configuration should I keep this method or it's better to remove it?